pkg/operator/sync.go

```diff
@@ -359,6 +391,56 @@ func newContainers(config *OperatorConfig, features map[string]bool) []corev1.Co
 	return containers
 }
 
+func newKubeProxyContainers(image string) []corev1.Container {
+	return []corev1.Container{
+		newKubeProxyContainer(image, "machineset-mtrc", metrics.DefaultMachineSetMetricsAddress, machineSetExposeMetricsPort),
+		newKubeProxyContainer(image, "machine-mtrc", metrics.DefaultMachineMetricsAddress, machineExposeMetricsPort),
+		newKubeProxyContainer(image, "nodelink-mtrc", metrics.DefaultNodeLinkMetricsAddress, nodelinkExposeMetricsPort),
+		newKubeProxyContainer(image, "mhc-mtrc", metrics.DefaultHealthCheckMetricsAddress, machineHealthCheckExposeMetricsPort),
+	}
+}
+
+func newKubeProxyContainer(image, portName, metricsPort string, exposePort int32) corev1.Container {
+	configMountPath := "/etc/kube-rbac-proxy"
+	TLSCerMountPath := "/etc/tls/private"
```

Comment:
I would rename this like so
```suggestion
	tlsCertMountPath := "/etc/tls/private"
```